### ISSUE TITLE
Draw master window border in different colour

### ISSUE THREAD

**Issue by proycon** [hyprwm/Hyprland#1265]
This is a minor feature request (very low priority) for the master layout; it might be an idea to optionally draw the border around the master window in different colour than the slave windows. Making the two easier to distinguish in situations where one has only two windows open (relevant after #1059 for users who actively use different orientations).

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
